app/herdbook.py

```diff
@@ -819,6 +819,13 @@ def update_certificate(i_number):
             uploaded = upload_certificate(
                 pdf_bytes=signed_data.getvalue(), ind_number=ind_data["number"]
             )
+            # Update breeding if litter size has changed
+            if breed_data.get("litter_size") != form.get("litter") or breed_data.get(
+                "litter_size6w"
+            ) != form.get("litter6w"):
+                breed_data.update(litter6w=form.get("litter6w"))
+                breed_data.update(litter=form.get("litter"))
```

Comment:
Didn't you refactor all the `litter`in the frontend to be called `litter-size` ? Here it's still `litter` and `litter6w`, would be good to double-check this.

Reply:
Probable i Messed up a rebase and stash

Reply:
Probably messed up with stash and rebase